runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingModeExecutionContext.java

```diff
@@ -591,7 +591,7 @@ public void flushState() {
               timerId,
               "",
               cleanupTime,
-              cleanupTime,
```

Comment:
We also seem to set GC timers in ReduceFnRunner.java. @kennknowles do you know why we have both?

Reply:
@reuvenlax  added in ReduceFnRunner.java